internal/cmd/run/cucumber.go

```diff
@@ -0,0 +1,178 @@
+package run
+
+import (
+	"os"
+
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/internal/backtrace"
+	"github.com/saucelabs/saucectl/internal/ci"
+	cmds "github.com/saucelabs/saucectl/internal/cmd"
+	"github.com/saucelabs/saucectl/internal/credentials"
+	"github.com/saucelabs/saucectl/internal/cucumber"
+	"github.com/saucelabs/saucectl/internal/docker"
+	"github.com/saucelabs/saucectl/internal/flags"
+	"github.com/saucelabs/saucectl/internal/framework"
+	"github.com/saucelabs/saucectl/internal/region"
+	"github.com/saucelabs/saucectl/internal/report/captor"
+	"github.com/saucelabs/saucectl/internal/saucecloud"
+	"github.com/saucelabs/saucectl/internal/segment"
+	"github.com/saucelabs/saucectl/internal/usage"
+	"github.com/saucelabs/saucectl/internal/viper"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+// NewCucumberCmd creates the 'run' command for cucumber.
+func NewCucumberCmd() *cobra.Command {
+	sc := flags.SnakeCharmer{Fmap: map[string]*pflag.Flag{}}
+
+	cmd := &cobra.Command{
+		Use:              "cucumber",
+		Short:            "Run Cucumber test",
+		Hidden:           true,
+		TraverseChildren: true,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			sc.BindAll()
+			return preRun()
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			// Test patterns are passed in via positional args.
+			viper.Set("suite::options::paths", args)
+
+			exitCode, err := runCucumber(cmd)
+			if err != nil {
+				log.Err(err).Msg("failed to execute run command")
+				backtrace.Report(err, map[string]interface{}{
+					"username": credentials.Get().Username,
+				}, gFlags.cfgFilePath)
+			}
+			os.Exit(exitCode)
+		},
+	}
+
+	sc.Fset = cmd.Flags()
+
+	sc.String("name", "suite::name", "", "Set the name of the job as it will appear on Sauce Labs.")
+	sc.String("platformName", "suite::platformName", "", "Run against this platform.")
+
+	// Cucumber
+	sc.String("cucumber-name", "suite::options::name", "", "Regular expressions of which scenario names should match one of to be run")
```

Comment:
A bit odd to call this `cucumber-name` no? Isn't `scenario` or `scenario-name` more apt?

Reply:
rename it as `scenario-name`